### PR TITLE
#507 feat(format bar): hide from post screen mobile only

### LINKED ISSUE #507: Make sure format bar on post screen is always sticky above the keyboard
MOBILE ONLY: Please make sure that in portrait view on mobile that the format bar is always sticky above the keyboard also please make the "enter" key be the post key. We will have no post button. 

post_portrait

Please note that if possible on landscape we don't want the format bar to be sticky. It gets in the way too much. 

post_landscape

### PR DESCRIPTION
**Pull request by danleyb2**
close #507

### CODE REVIEW

File: front/pages/ideas/create.vue [hunk 1]
```diff
@@ -301,6 +301,9 @@ export default {
       position: fixed;
       bottom: 0;
     }
+    @media (max-width: $screen-md-min) {
```
Comment (aldarund) [inline front/pages/ideas/create.vue:304]: there is a vuetify class for this. hidden-md-and-down hidden-md-and-up etc etc

**Comment by vedmant** [comment]
lgtm, just one element is hidden on mobile

File: front/pages/ideas/create.vue (hunk 1 above)
Comment (vedmant) [reply front/pages/ideas/create.vue:304]: @danleyb2 could you make this edit?

Comment (danleyb2) [reply front/pages/ideas/create.vue:304]: @aldarund @vedmant This style targets `trix-toolbar` it's a child componet of `components/TrixWrapper.vue` and so there is no way of adding vuetify classes directly

**Pull request merged**